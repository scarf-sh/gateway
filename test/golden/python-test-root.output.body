<!DOCTYPE html><html><body>
<a href="/simple/cipy/">cipy</a><br>
<a href="/simple/numpy/">numpy</a><br>
</body></html>