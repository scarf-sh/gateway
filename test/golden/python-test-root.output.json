{
    "capture": "PythonCapture {pythonCapturePackage = Nothing, pythonCaptureVersion = Nothing, pythonCaptureFileName = Nothing, pythonCaptureFileBackendURL = Nothing}",
    "headers": {
        "Content-Length": "118",
        "Content-Type": "text/html",
        "Etag": "\"agnhjMSIWv0_F5FCrBDmit2jdkTb4QLM7v4tnUKUs-U=\""
    },
    "status": 200
}